extractors/other/simple-aws-s3/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Simple AWS S3
+permalink: /extractors/other/simple-aws-s3/
+---
+
+* TOC
+{:toc}
+
+This extractor loads a single or multiple CSV files from AWS S3 and stores them in a single table in Storage.
```

Comment:
zduraznil bych to single table, chybi mi tam neco jako "pokud vam staci nahravat do jedny tabulky, tak tohle je pro vas, jinak ten druhy" a to same u toho druheho

Reply:
```
For a more complex AWS S3 extractor allowing you to use multiple S3 buckets and multiple tables in Storage please 
see [AWS S3 Extractor](/extractors/other/aws-s3).
```
?